lib/src/main/java/graphql/nadel/schema/NeverWiringFactory.kt

```diff
@@ -100,18 +100,31 @@ open class NeverWiringFactory : WiringFactory {
     }
 
     override fun providesDataFetcher(environment: FieldWiringEnvironment): Boolean {
-        return true
+        return false
```

Comment:
So we dont create a DF per field or make a CodeRegistry entry per field as well but rather use a default